pkg/feature/builder.go

```diff
@@ -67,9 +67,11 @@ func (fb *featureBuilder) Source(source featurev1.Source) *featureBuilder {
 }
 
 // TargetNamespace sets the namespace in which the feature should be applied.
-// If not set, the feature will be applied in the application namespace.
+// Calling it multiple times in the builder chain will have no effect, as the first value is used.
```

Comment:
I think the original comments is still valid, but with this new one to explain the case "if it has been set"

Reply:
Not on this level, no. If not set it will fail. Implicitly setting the target namespace before building feature is done in the handler, but that's different.